course/steps/landing/en.md

```diff
@@ -46,8 +46,30 @@ The query schema is one of the [custom query search result props](https://vtex.i
   - Sorts by latest release date;
   - Hides unavailable items;
   - Displays a max of 8 items per page;
-  - Uses "Blue Top Retro Camera" as *query*.
-
+  - Uses "Camera" as *query*.
+
+7. At this point, you're probably not seeing the block on the landing page. This is due to the fact that we have not add a block to the search-result-layout.customQuery yet. You can do that by adding the following lines:
+    ```diff
+        // store/blocks/search-landing.jsonc
+        {
+            ...
+            "search-result-layout.customQuery": {
+                "props": {
```

Comment:
In this language the indentation is wrong

Reply:
I've just fixed